.github/workflows/e2e-versions.yml

```diff
@@ -20,8 +20,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        distribution: ['temurin', 'adopt', 'adopt-openj9', 'zulu', 'liberica'] # internally 'adopt-hotspot' is the same as 'adopt'
+        distribution: ['temurin', 'adopt', 'adopt-openj9', 'zulu', 'liberica', 'microsoft' ] # internally 'adopt-hotspot' is the same as 'adopt'
         version: ['8', '11', '16']
```

Comment:
Can we replace 16 with 17 here, since 17 is the latest LTS now?

Reply:
Probably worth doing in a different PR since it is unrelated to adding MSFT.

Reply:
Ah, yes. That'd be better indeed.